packages/patternfly-react/src/components/VerticalNav/VerticalNavItemHelper.js

```diff
@@ -80,11 +80,17 @@ class BaseVerticalNavItemHelper extends React.Component {
     updateNavOnItemBlur(primary, secondary, tertiary, this.idPath(), idPath, noDelay, onBlur);
   };
 
+  eventModified = event => !!(event.metaKey || event.altKey || event.ctrlKey || event.shiftKey);
+
   onItemClick = event => {
     const { primary, secondary, tertiary } = this.getContextNavItems();
     const { isMobile, preventHref, updateNavOnItemClick, idPath } = this.props;
     const { onClick } = this.navItem();
 
+    if (this.eventModified(event)) {
```

Comment:
instead of this can you not just set the `preventHref` prop to false?

Reply:
Setting `preventHref` to false does not take care of the issue entirely. While it opens the link in new tab, it also changes location of current tab because the`onItemClick` handler continues its execution.

Reply:
Right, we wrote this component to do exactly that. Can you add a new `preventUpdate` prop to do this instead? Consumers may wish for modifier keys like shift and alt to do other things.

This first lines of `onItemClick` could just look like:
```tsx
const { isMobile, preventHref, updateNavOnItemClick, idPath, preventUpdate } = this.props;
if (preventUpdate) {
  return;
}
```
and you could pass `preventUpdate` instead of `preventHref`.